### PR TITLE
- Update to install superset 1.0

### PR DESCRIPTION
**Pull request by ultraheroe**
### Description of the Change

Updating to install and use the new 1.0 Superset features
   - New vars for SIP-15, emailing, chrome, etc
   - Install chrome and chromedriver for reports
   - Some new tasks and checks
   - superset_config.py template updated

### Benefits

Install and configure some of the new features of the 1.0 superset version

### Possible Drawbacks

Some existing dashboards, roles or users may be corrupted or deleted

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
 ## [Unreleased](https://github.com/idealista/superset_role/tree/develop)
+- Install new 1.0 superset version
```
Comment (blalop) [inline CHANGELOG.md:6]: We usually use a specific format for changelogs - see https://github.com/idealista/timescaledb_role/blob/main/CHANGELOG.md

File: README.md [hunk 2]
```diff
@@ -30,15 +30,15 @@ Ansible 2.9.x.x version installed.
...
+For testing purposes, [Molecule](https://molecule.readthedocs.io/) with [Docker](https://www.docker.com/) as driver and [Goss] (<https://github.com/aelsabbahy/goss>) as verifier.
 
 ### Installing
 
 Create or add to your roles dependency file (e.g requirements.yml):
 
 ```yml
 - src: idealista.superset_role
-  version: 1.0.1
+  version: develop
```
Comment (blalop) [inline README.md:41]: Better use a release version instead of a branch :)

File: defaults/main.yml [hunk 3]
```diff
@@ -4,16 +4,21 @@
...
 # Set true to force the download and installation of the binary
 superset_force_reinstall: false
 
+# Owner
+superset_user: superset
+superset_group: superset
+
 # Installation path
 superset_app_path: "/opt/{{ superset_app_name }}"
+superset_user_home: "/home/{{ superset_user }}"
```
Comment (blalop) [inline defaults/main.yml:21]: Is this var used?

File: molecule/default/Dockerfile.j2 [hunk 4]
```diff
@@ -13,4 +13,5 @@ RUN apt-get update && \
 
 EXPOSE 8088/tcp
```
Comment (blalop) [inline molecule/default/Dockerfile.j2:14]: All this ports should go under exposed_ports in molecule.yml file

**Review by blalop — CHANGES_REQUESTED** [review]

**Review by frantsao — APPROVED** [review]

File: defaults/main.yml (hunk 3 above)
Comment (ultraheroe) [reply defaults/main.yml]: app home updated

File: molecule/default/Dockerfile.j2 (hunk 4 above)
Comment (ultraheroe) [reply molecule/default/Dockerfile.j2]: done

File: CHANGELOG.md (hunk 1 above)
Comment (ultraheroe) [reply CHANGELOG.md:42]: updated changelog

**Review by ultraheroe — COMMENTED** [review]
Re reviewed

**Review by caldito — APPROVED** [review]

File: Pipfile [hunk 5]
```diff
@@ -6,8 +6,8 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-ansible = "==2.9.9"
-molecule = "==3.0.4"
+ansible = "==2.9.21"
+molecule = "==3.0.6"
```
Comment (blalop) [inline Pipfile:10]: Why not molecule 3.0.8 instead?

File: molecule/default/molecule.yml [hunk 6]
```diff
@@ -24,12 +24,14 @@ platforms:
     stop_signal: 'RTMIN+3'
     exposed_ports:
       - 8088/tcp
-      - 8081/tcp
+      - 8080/tcp
       - 5000/tcp
+      - 5555/tcp
     published_ports:
```
Comment (blalop) [inline molecule/default/molecule.yml:30]: Should we publish the minimal amount of ports possible?

**Review by blalop — APPROVED** [review]

File: molecule/default/molecule.yml (hunk 6 above)
Comment (ultraheroe) [reply molecule/default/molecule.yml:29]: yeah, I've removed the unnecessary

File: Pipfile (hunk 5 above)
Comment (ultraheroe) [reply Pipfile]: Right! I've upgraded to 3.0.8 without errors

**Review by emepege — APPROVED** [review]

**Pull request merged**